includes/sidebar-functions.php

```diff
@@ -142,3 +142,69 @@ function today_display_sidebar_menu( $args=array() ) {
 
 	return today_display_sidebar_content( $title, $content );
 }
+
+
+/**
+ * Returns the sidebar markup for pages that utilize a customizable sidebar.
+ *
+ * @since 1.0.0
+ * @author Cadie Brown
+ * @param integer $post_id WP Post ID
+ * @return string HTML markup for the sidebar
+ **/
+function today_get_sidebar_markup( $post_id ) {
+	$markup = '';
+
+	if ( have_rows( 'sidebar_content', $post_id ) ) {
+		while ( have_rows( 'sidebar_content', $post_id ) ) : the_row();
+			switch ( get_row_layout() ) {
+				case 'sidebar_events' :
+					$feed_url  = get_sub_field( 'events_feed_url' ) ?: 'https://events.ucf.edu/upcoming/feed.json';
```

Comment:
Might make sense to make these customizer settings in a future update. Just in case we need to be able to update the default.

Reply:
Updated to remove hard-coded events URLs here and in the ACF config.  When a custom feed URL or View All URL are not provided, the URLs will be determined by the UCF Events plugin's settings/defaults.